cassandra/cassandra_test.go

```diff
@@ -0,0 +1,441 @@
+package cassandra
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/gocql/gocql"
+	"github.com/stretchr/testify/require"
+	"github.com/testcontainers/testcontainers-go"
+	cassandracontainer "github.com/testcontainers/testcontainers-go/modules/cassandra"
+)
+
+const (
+	// cassandraImage is the default image used for running cassandra in tests.
+	cassandraImage              = "cassandra:latest"
+	cassandraImageEnvVar string = "TEST_CASSANDRA_IMAGE"
+	cassandraPort               = "9042/tcp"
+)
+
+// newTestStore creates a Cassandra container using the official module
+func newTestStore(t testing.TB) string {
```

Comment:
suggestion: if we are going to always create the same Store and Config in the callers of this function, I'd return here the Store instead. This is done in that way in the other storages using testcontainers, so I'd follow the same pattern.